docker/lfmerge/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:bionic
+
+RUN apt update && apt install curl \
+&& curl -L http://linux.lsdev.sil.org/downloads/sil-testing.gpg > /etc/apt/trusted.gpg.d/sil-testing.gpg \
+&& echo "deb http://linux.lsdev.sil.org/ubuntu bionic main" > /etc/apt/sources.list.d/linux-lsdev-sil-org.list \
+&& apt update
+
+RUN apt install -y lfmerge
```

Comment:
I was curious about the `apt` usage I see...I'm used to seeing `apt-get`...I'm "guessing" they are probably the same but one is meant to be interactive while another is meant be used programmatically?